src/test/storage/value_segment_test.cpp

```diff
@@ -54,4 +54,26 @@ TEST_F(StorageValueSegmentTest, MemoryUsage) {
   EXPECT_EQ(int_value_segment.estimate_memory_usage(), size_t{8});
 }
 
+TEST_F(StorageValueSegmentTest, SubscriptOperator) {
+  int_value_segment.append(1337);
+  int_value_segment.append(1338);
+  EXPECT_EQ(int_value_segment[0], AllTypeVariant(1337));
+  EXPECT_EQ(int_value_segment[1], AllTypeVariant(1338));
+}
+
+TEST_F(StorageValueSegmentTest, Values) {
+  int_value_segment.append(1337);
+  int_value_segment.append(1338);
+
+  auto values = int_value_segment.values();
+  std::vector<int> expected_variants = {1337, 1338};
```

Comment:
The opossum style guide recommends using auto-to-stick:

```suggestion
  auto expected_variants = std::vector<int>{1337, 1338};
```

Reply:
Valid point. I'll integrate your change.